builder/common/run_config.go

```diff
@@ -390,11 +411,77 @@ type RunConfig struct {
 	// subnet-12345def, where Packer will launch the EC2 instance. This field is
 	// required if you are using an non-default VPC.
 	SubnetId string `mapstructure:"subnet_id" required:"false"`
-	// [Tenancy](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/dedicated-instance.html) used
-	// when Packer launches the EC2 instance, allowing it to be launched on dedicated hardware.
+	// The license configurations.
```

Comment:
Should we describe the default behavior too here ?

It used to be `The default is "default", meaning shared tenancy. Allowed values are "default", "dedicated" and "host".`

So we could say: `By default, when this setting is unspecified, shared tenancy will be selected.` or something